src/style.css

```diff
@@ -0,0 +1,117 @@
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+
+}
+
+.BigForm {
+  margin-top: 20px;
+  margin-bottom: 20px;
+
+}
+
+.iconBig {
+  position: absolute;
+  box-sizing: border-box;
+  margin-top: 25px;
+  margin-left: 26px;
+  width: 19px;
+
+  height: 19px;
+  
+}
+
+.search-bar {
+  position: relative;
+
+}
+
+.search-1 {
+  padding-top: 28px;
+  padding-left: 62px;
+  padding-bottom: 27px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  width: 100%;
+
+}
```

Comment:
get rid of this empty line at the end of class block

Reply:
the point is that it does not allow to commit. Try to make empty lines